include/zephyr/bluetooth/l2cap.h

```diff
@@ -579,22 +579,20 @@ int bt_l2cap_chan_disconnect(struct bt_l2cap_chan *chan);
  *  size the buffers for the for the outgoing buffer pool.
  *
  *  When sending L2CAP data over an LE connection the application is sending
- *  L2CAP SDUs. The application can optionally reserve
+ *  L2CAP SDUs. The application should reserve
```

Comment:
When providing suggestions (like `should`) I find it helpful to also explain _why_. 

Why should the application reserve this? What happens if it does/doesn't?

The migration guide now also states that it is required, but that does not fit with "should".

Reply:
@jori-nordic Should this be changed to a `shall`?